.travis.yml

```diff
@@ -11,7 +11,7 @@ matrix:
     - env: TEST_COMMAND="make test-venv && CHPL_HOME=$PWD ./util/run-in-venv.bash ./util/test/check_annotations.py"
       git:
         depth: 100000
-    - env: TEST_COMMAND="./util/devel/lookForBadAlloc"
+    - env: TEST_COMMAND="./util/devel/lookForBadRTCalls"
```

Comment:
I might have called it `lookForBadCallsInRT` instead of `lookForBadRTCalls` since the latter sounds like we're looking for bad calls *to* the runtime instead of bad calls *in* it, but I'm okay if you leave it as is.